components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -462,6 +505,33 @@ func (c *certRotateFlow) certRotateOS(sshUtil SSHUtil, certs *certificates, infr
 	return nil
 }
 
+func (c *certRotateFlow) getSkipIpsListForOsRootCACNPatching(infra *AutomateHAInfraDetails, sshUtil SSHUtil, certs *certificates, nodesCn string, flagsObj *certRotateFlags) ([]string, error) {
+
+	//fetching current config from automate
+	c.pullConfigs.setInfraAndSSHUtil(infra, sshUtil)
+	//fetching current config from automate
+	automateCurrentConfig, err := c.pullConfigs.pullAutomateConfigs()
+	if err != nil {
+		return nil, err
+	}
+
+	// Patching root-ca to frontend-nodes for maintaining the connection.
+	cn := nodesCn
```

Comment:
No need of creating new variable pass nodesCn directly

Reply:
done